### PR TITLE
Fixed reachability on internal assets

### PR DESCRIPTION
**Pull request by Pastea**
Fixed old references and logic bug

### CODE REVIEW

File: secbox [hunk 1]
```diff
@@ -289,15 +289,15 @@ suse_internal_network() {
...
+        "10.144.39.65/22"
         # dns lookup can introduce a very long delay when the dns server is not
         # reachable, so I test domain name resolution only if the above worked
-        "wotan.suse.de/22"
+        "login.suse.de/22"
     )
 
     for _address in "${_know_internal_addresses[@]}"; do
         # Fail at first connection that can't be estabilished
-        timeout 1 sh -c "echo > /dev/tcp/${_address}" >/dev/null 2>&1 || return
```
Comment (StayPirate) [inline secbox:300]: I think that `||` is the correct one. The for is intended to be read as **if all** and not **if any**.

**Comment by StayPirate** [comment]
Switching from `wotan.suse.de` to `login.suse.de` is a great idea and will fix the `suse_internal_network()` check.

Unfortunately `secbox` also uses wotan to provide remote access for a couple of shares: `/mount` and `/suse`.  In turn, these are used along with some embedded tools which expect to find mounted remote folders at those locations. Or at least that was the case for some time.
I first want to reconsider if `--sshfs` and `--nfs` are still required in secbox or if we can get rid of them and simplify the script.

File: secbox (hunk 1 above)
Comment (Pastea) [reply secbox:300]: Yeah, you are right, I missunderstanded the flow.

**Comment by StayPirate** [comment]
If you want you can test [this version](https://github.com/StayPirate/secbox/tree/remove-sshfs) which gets rid of the remote resource. I'm testing it locally and it seems to work smoothly.

**Comment by Pastea** [comment]
Yeah, I confirm, it works greatly. Thanks.

**Pull request merged**